Resources/Info.plist

```diff
@@ -60,5 +50,15 @@
 			<string>safemode</string>
 		</dict>
 	</array>
+	<key>UIRequiredDeviceCapabilities</key>
```

Comment:
Am I missing something or does this change just move keys down and not change anything?